### PR TITLE
Move to serde_codegen

### PR DESCRIPTION
**Pull request by ghost**
serde_macros was using libsyntax, which seems to be changing quite often, requiring to always get the specific version matching your compiler version.

This was impractical (and broke my build), so I decided to try the "stable" way of doing it, which revealed to be relatively easy.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#![feature(custom_derive, plugin)]
-#![plugin(serde_macros)]
+//#![feature(custom_derive, plugin)]
+//#![plugin(serde_macros)]
```
Comment (Nemikolh) [inline src/lib.rs]: Maybe you could get rid of those. :3

**Comment by Nemikolh** [comment]
Nice! :tada:

**Comment by ghost** [comment]
Do we really need Cargo.lock in this repository?

**Comment by Nemikolh** [comment]
It's recommended for binary to get reproducible builds. So I think it make a lot of sense to have one on both `sarosa` and `lycan`.

**Comment by ghost** [comment]
So, can I consider this as "reviewed" and merge it?

**Comment by Nemikolh** [comment]
Yeap, this looks good to me!

**Comment by ghost** [comment]
Ninja!

**Pull request merged**